src/TypeLevel/DataSchema.purs

```diff
@@ -0,0 +1,238 @@
+module TypeLevel.DataSchema
```

Comment:
I'd personally change the namespace of the module from `TypeLevel` to `Plutus`, or even `Plutus.Types` (so possible `Plutus.Types.DataSchema`), just to make it clear that this is quite Plutus-specific

Reply:
Done